### ISSUE TITLE
Suggestions for improving the presentation

### ISSUE THREAD

**Issue by stefanwouldgo** [storm-org/storm-spec#4]
Thank you for this proposal. I've been thinking about similar schemes (paying for storage with pure bitcoin in a trustless manner) for a long time. I think your proposal has the potential to solve the CS theory part of the problem at least. I'm not sure about the economic aspects. 

Anyway, I quite like the presentation but believe it can be made even clearer. For example, I would avoid mentioning the core concepts and especially the PCP concept, because IMHO it suggests that one would need to utilize or even understand the general PCPs from Theoretical Computer Science, whereas here we have a very particular ZKP for a specific problem.

**Comment by dr-orlovsky** [comment]
Agree, I will include this into the new release of the protocol description
